### YamlMime:ManagedReference
items:
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException
  id: EntityDataSourceValidationException
  children:
  - System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor
  - System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String)
  - System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String,System.Exception)
  - System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions
  langs:
  - csharp
  name: EntityDataSourceValidationException
  nameWithType: EntityDataSourceValidationException
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException
  type: Class
  summary: "Representa erros que ocorrem ao validar as propriedades de uma fonte de dados dinâmicos."
  syntax:
    content: 'public sealed class EntityDataSourceValidationException : Exception, System.Web.DynamicData.IDynamicValidatorException'
  inheritance:
  - System.Exception
  implements:
  - System.Web.DynamicData.IDynamicValidatorException
  inheritedMembers: []
  platform:
  - net462
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor
  id: '#ctor'
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  langs:
  - csharp
  name: EntityDataSourceValidationException()
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException()
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException()
  type: Constructor
  assemblies:
  - System.Web.Entity
  namespace: System.Web.UI.WebControls
  summary: "Cria uma nova instância do <xref href=&quot;System.Web.UI.WebControls.EntityDataSourceValidationException&quot;> </xref> classe com uma mensagem fornecida pelo sistema que descreve o erro."
  remarks: "Este construtor inicializa o <xref:System.Exception.Message%2A>propriedade da nova instância para uma mensagem fornecida pelo sistema que descreve o erro, como &quot;DefaultMessageDisplayedByParameterlessCtorWriterMustSupply&quot;.</xref:System.Exception.Message%2A> Esta mensagem leva em conta a cultura atual do sistema.       A tabela a seguir mostra os valores de propriedade inicial para uma instância de <xref:System.InsufficientMemoryException>.</xref:System.InsufficientMemoryException>      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| A cadeia de caracteres de mensagem de erro localizada. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public EntityDataSourceValidationException ();
    parameters: []
  overload: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  langs:
  - csharp
  name: EntityDataSourceValidationException(String)
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException(String)
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException(String)
  type: Constructor
  assemblies:
  - System.Web.Entity
  namespace: System.Web.UI.WebControls
  summary: "Cria uma nova instância do <xref href=&quot;System.Web.UI.WebControls.EntityDataSourceValidationException&quot;> </xref> classe com uma mensagem especificada que descreve o erro."
  remarks: "A tabela a seguir mostra os valores de propriedade inicial para uma instância de <xref:System.Web.UI.WebControls.EntityDataSourceValidationException>.</xref:System.Web.UI.WebControls.EntityDataSourceValidationException>      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| A cadeia de caracteres de mensagem de erro especificada no `message`. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public EntityDataSourceValidationException (string message);
    parameters:
    - id: message
      type: System.String
      description: "A mensagem que descreve a exceção. O chamador desse construtor é necessário para garantir que essa cadeia de caracteres foi localizada para a cultura atual do sistema."
  overload: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  langs:
  - csharp
  name: EntityDataSourceValidationException(String,Exception)
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException(String,Exception)
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException(String,Exception)
  type: Constructor
  assemblies:
  - System.Web.Entity
  namespace: System.Web.UI.WebControls
  summary: "Cria uma nova instância do <xref href=&quot;System.Web.UI.WebControls.EntityDataSourceValidationException&quot;> </xref> classe com uma mensagem de erro especificada e uma referência à exceção interna que é a causa dessa exceção."
  remarks: "Uma exceção que é lançada como um resultado direto de uma exceção anterior deve incluir uma referência à exceção anterior no <xref:System.Exception.InnerException%2A>propriedade.</xref:System.Exception.InnerException%2A> O <xref:System.Exception.InnerException%2A>propriedade retorna o mesmo valor que é passado para o construtor, ou `null` se o <xref:System.Exception.InnerException%2A>propriedade não fornecer o valor da exceção interna para o construtor.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       A tabela a seguir mostra os valores de propriedade inicial para uma instância de <xref:System.Web.UI.WebControls.EntityDataSourceValidationException>.</xref:System.Web.UI.WebControls.EntityDataSourceValidationException>      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| A cadeia de caracteres de mensagem de erro especificada no `message`. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public EntityDataSourceValidationException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "A mensagem que descreve a exceção. O chamador desse construtor é necessário para garantir que essa cadeia de caracteres foi localizada para a cultura atual do sistema."
    - id: innerException
      type: System.Exception
      description: "A exceção que é a causa da exceção atual. Se o `innerException` parâmetro não é <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, a exceção atual é gerada em um <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> bloco que manipula a exceção interna."
  overload: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions
  id: System#Web#DynamicData#IDynamicValidatorException#InnerExceptions
  isEii: true
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  langs:
  - csharp
  name: System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  nameWithType: EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  type: Property
  assemblies:
  - System.Web.Entity
  namespace: System.Web.UI.WebControls
  summary: "Obtém as exceções que ocorrem quando um campo de dados novos ou editados é validado."
  remarks: "Esse membro é uma implementação de membro de interface explícita. Ele pode ser usado somente quando o <xref:System.Web.UI.WebControls.EntityDataSourceValidationException>instância é convertida em um <xref:System.Web.DynamicData.IDynamicValidatorException>interface.</xref:System.Web.DynamicData.IDynamicValidatorException> </xref:System.Web.UI.WebControls.EntityDataSourceValidationException>"
  syntax:
    content: System.Collections.Generic.IDictionary<string,Exception> System.Web.DynamicData.IDynamicValidatorException.InnerExceptions { get; }
    return:
      type: System.Collections.Generic.IDictionary{System.String,System.Exception}
      description: "Uma coleção que contém todas as exceções que ocorreram."
  overload: System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: EntityDataSourceValidationException()
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException()
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException()
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String)
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: EntityDataSourceValidationException(String)
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException(String)
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String,System.Exception)
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: EntityDataSourceValidationException(String,Exception)
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException(String,Exception)
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException(String,Exception)
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  nameWithType: EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
- uid: System.Collections.Generic.IDictionary{System.String,System.Exception}
  parent: System.Collections.Generic
  isExternal: true
  name: IDictionary<String,Exception>
  nameWithType: IDictionary<String,Exception>
  fullName: System.Collections.Generic.IDictionary<System.String,System.Exception>
  spec.csharp:
  - uid: System.Collections.Generic.IDictionary`2
    name: IDictionary
    nameWithType: IDictionary
    fullName: IDictionary<System.String,System.Exception>
  - name: <
    nameWithType: <
    fullName: <
  - uid: System.String
    name: String
    nameWithType: String
    fullName: String
  - name: ','
    nameWithType: ','
    fullName: ','
  - uid: System.Exception
    name: Exception
    nameWithType: Exception
    fullName: Exception
  - name: '>'
    nameWithType: '>'
    fullName: '>'
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor*
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: EntityDataSourceValidationException
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions*
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  nameWithType: EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
